Doc/whatsnew/3.10.rst

```diff
@@ -596,6 +602,30 @@ Changes in the Python API
   a 16-bit unsigned integer.
   (Contributed by Erlend E. Aasland in :issue:`42393`.)
 
+* The ``loop`` parameter to nearly all :mod:`asyncio` functions has been removed
+  following deprecation in Python 3.8. For example, a coroutine that currently
+  look like this::
+
+     import asyncio
+
+     async def foo(loop):
+         await asyncio.sleep(loop=loop)
+
+  Can *usually* be replaced with this::
+
+     import asyncio
+
+     async def foo(loop):
+         await asyncio.sleep()
+
+  Maintaining backwards compatibility with 3.6 code is slightly trickier.
+  Previous code using :mod:`asyncio` functions and classes, passing the loop
+  explicitly and relying on special behavior will error on 3.10.  A possible
+  option is to guard the old call with a check for Python versions before 3.7
+  and use the new call signature for newer versions.
```

Comment:
It's possible to call sleep(), without passing a loop argument, in Python 3.6.

Extract of Python 3.6 implementation:

```
@coroutine
def sleep(delay, result=None, *, loop=None):
    ...
    if loop is None:
        loop = events.get_event_loop()
    ...
```

Reply:
Sorry, the language here wasn't too clear, what I meant was that code _relying specifically_ on the ability to attach asyncio futures to whatever loop they choose will break. Maybe I should just remove it to reduce confusion.